### ISSUE TITLE
import org.papervision3d.objects.Plane not found with components 1.5.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [paazmaya/papervision3d#14]
```
What steps will reproduce the problem?
1. Just try to import org.papervision3d.objects.Plane with the components 
1.5.1 , it's not working , not found.  Work when you download the classes 
package you offer but not with the components.. maybe something missing in 
it?

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
1.5.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `kameleon...@gmail.com` on 27 Jul 2007 at 11:31

**Comment by GoogleCodeExporter** [comment]
```
thanks for letting me know, I'll check to see, but you're probably right that 
it's
not included in the component's package.


```

Original comment by `neori...@gmail.com` on 27 Jul 2007 at 1:17
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
are you planing a new release with the missing classes included? ;) by the way, 
the 
engine is incredible. I just love it so much
```

Original comment by `kameleon...@gmail.com` on 28 Jul 2007 at 4:33

**Comment by GoogleCodeExporter** [comment]
```
Fixed, correct  ? 
```

Original comment by `r.hauw...@gmail.com` on 8 Apr 2008 at 11:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Version 2.0 which isn't released yet
```

Original comment by `neori...@gmail.com` on 9 Apr 2008 at 11:47
- Changed state: **Fixed**
